javatools/src/main/java/org/xvm/asm/Component.java

```diff
@@ -3417,7 +3417,7 @@ protected void registerConstants(ConstantPool pool)
                     TypeConstant   type      = entry.getValue();
 
                     mapNew.put((StringConstant) pool.register(constName),
-                               (TypeConstant)   (type == null ? null : pool.register(type)));
+                               (TypeConstant) (type == null ? null : pool.register(type)));
                     }
                 m_mapParams = mapNew;
```

Comment:
The assertion in this file (line 543) with the list assignment has side effects. Is this intentional? 

There are a few raw times in this changed code, and other linting issues. Perhaps we can start getting rid of those kinds of findings when we do bug fixes in their vicinity, for example unlinkSibling takes a raw map and so on.

Reply:
I don't know what assertion you're referring to. There is none on line 543 at Component.java